cpp/src/parquet/properties.h

```diff
@@ -941,6 +942,18 @@ class PARQUET_EXPORT ArrowReaderProperties {
     return coerce_int96_timestamp_unit_;
   }
 
+  /// Enable Parquet supported Arrow ExtensionTypes.
+  ///
+  /// When enabled, Parquet will use supported Arrow ExtensionTypes by mapping correctly
+  /// mapping them to Arrow types at read time. Currently only arrow::extension::json()
+  /// extension type is supported. Columns whose LogicalType is JSON will be interpreted
+  /// as arrow::extension::json() ExtensionType with storage type utf8, large_utf8 or
```

Comment:
Is it possible for users to pick one from `utf8, large_utf8 or utf8_view` on read?

Reply:
I don't think it makes sense to expose such a setting specifically for JSON (rather than generically for all string-like columns). Also, if you store the Arrow schema, the original storage type should be restored (but that's not implemented for `utf8_view` AFAICT).

Reply:
@wgtmac do you mean cast on read?

Reply:
Yes, I think this is a more general question for all string types.